### PR TITLE
New util `findAnimationName` and some fixes.

### PR DESCRIPTION
**Pull request by evilebottnawi**
/cc @davidtheclark @jeddy3

### CODE REVIEW

File: src/rules/no-unknown-animations/index.js [hunk 1]
```diff
@@ -37,20 +34,14 @@ export default function (actual) {
...
-          if (animationShorthandKeywords.has(value.toLowerCase())) { continue }
-          // Ignore functions
-          if (value.indexOf("(") !== -1) { continue }
-          checkAnimationName(value, decl, decl.value.indexOf(value))
-        }
+        const animationNames = findAnimationName(decl.value)
+        if (animationNames.length === 0) { return }
+
+        animationNames.forEach((animationNameNode) => {
+          if (basicKeywords.has(animationNameNode.value)) { return }
```
Comment (davidtheclark) [inline src/rules/no-unknown-animations/index.js]: Does this need to `toLowerCase()` the value?

Comment (evilebottnawi) [inline src/rules/no-unknown-animations/index.js]: @davidtheclark yep

**Comment by davidtheclark** [comment]
LGTM.

@evilebottnawi in the future when you add other fixes to a PR it's helpful for reviewing if they're explained a little more than "other fixes" :)

**Comment by evilebottnawi** [comment]
@davidtheclark I'll go to sleep, and tomorrow we will close https://github.com/stylelint/stylelint/issues/1101

**Comment by jeddy3** [comment]
Nice. I think we we'll be able to do a release tomorrow then.

Good work this weekend everyone :)

**Pull request merged**